commons-security/src/main/kotlin/ebi/ac/uk/security/service/SecurityService.kt

```diff
@@ -2,84 +2,52 @@ package ebi.ac.uk.security.service
 
 import ac.uk.ebi.biostd.persistence.model.User
 import ac.uk.ebi.biostd.persistence.repositories.UserDataRepository
-import arrow.core.Option
-import com.fasterxml.jackson.databind.ObjectMapper
-import ebi.ac.uk.security.integration.model.SignUpRequest
-import ebi.ac.uk.security.model.TokenPayload
-import io.jsonwebtoken.JwtParser
-import io.jsonwebtoken.Jwts
-import io.jsonwebtoken.MalformedJwtException
-import io.jsonwebtoken.SignatureAlgorithm
-import io.jsonwebtoken.SignatureException
-import mu.KotlinLogging
-import java.security.MessageDigest
-import java.util.Arrays
+import ebi.ac.uk.api.security.LoginRequest
+import ebi.ac.uk.api.security.RegisterRequest
+import ebi.ac.uk.security.exception.UserNotFoundException
+import ebi.ac.uk.security.model.UserInfo
+import ebi.ac.uk.security.util.PasswordVerifier
+import ebi.ac.uk.security.util.TokenUtil
 import java.util.UUID
 
-private val logger = KotlinLogging.logger {}
-
 class SecurityService(
-    private val jwtParser: JwtParser,
-    private val objectMapper: ObjectMapper,
     private val userRepository: UserDataRepository,
-    private val tokenHash: String,
-    private val sha1: MessageDigest = MessageDigest.getInstance("SHA1")
+    private val passwordVerifier: PasswordVerifier,
+    private val tokenUtil: TokenUtil
 ) {
 
-    fun fromToken(token: String): Option<User> {
-        var tokenUser = Option.empty<TokenPayload>()
+    fun login(loginRequest: LoginRequest): UserInfo {
+        val user = userRepository.findByLoginOrEmail(loginRequest.login, loginRequest.login)
```

Comment:
```
val user = userRepository
        .findByLoginOrEmail(loginRequest.login, loginRequest.login)
        .orElseThrow { throw UserNotFoundException(loginRequest.login) }
```

Since we use a lot of functions, we could adopt this statndard of identing by operations. WDYT?

Reply:
look much better